kmir/k-src/mir-statement.md

```diff
@@ -1,5 +1,5 @@
 ```k
-requires "mir-rvalue.md"
+requires "mir-rvalue-syntax.md"
```

Comment:
It looks like `requires "mir-place.md"` can also be removed (mir-rvalue-syntax.md requires mir-operand.md requires mir-pace.md).

Reply:
Thanks for this. I will remove it. I am not 100% committed to having only required imports vs. all dependent imports---I can see it going both way.

For now, I'll commit this since I think the other PR changes are good. We can add back in these imports in a later version if we decide that going that route is better.